addons/sale_management/models/sale_order_line.py

```diff
@@ -19,7 +19,9 @@ def _compute_name(self):
                 for template_line in line.order_id.sale_order_template_id.sale_order_template_line_ids:
                     if line.product_id == template_line.product_id:
                         lang = line.order_id.partner_id.lang
-                        line.name = template_line.with_context(lang=lang).name + line.with_context(lang=lang)._get_sale_order_line_multiline_description_variants()
+                        # If the template's name is empty, use the product description
```

Comment:
No need for the fallback, don't enter the if when there is no name on the template line